chpl/chpl-service/src/main/java/gov/healthit/chpl/realworldtesting/domain/RealWorldTestingReport.java

```diff
@@ -0,0 +1,64 @@
+package gov.healthit.chpl.realworldtesting.domain;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@AllArgsConstructor
+public class RealWorldTestingReport {
+    private String acbName;
+    private String chplProductNumber;
+    private String productName;
+    private Long productId;
+    private String developerName;
+    private Long developerId;
+    private Integer rwtEligibilityYear;
+    private String rwtPlansUrl;
+    private LocalDate rwtPlansCheckDate;
+    private String rwtResultsUrl;
+    private LocalDate rwtResultsCheckDate;
+    private String rwtPlansMessage;
+    private String rwtResultsMessage;
+
+    public List<String> toListOfStrings() {
+        List<String> results = new ArrayList<String>();
+        results.add(acbName);
+        results.add(chplProductNumber);
+        results.add(productName);
+        results.add(productId == null ? null : productId.toString());
+        results.add(developerName);
+        results.add(developerId == null ? null : developerId.toString());
+        results.add(rwtEligibilityYear == null ? null : rwtEligibilityYear.toString());
+        results.add(rwtPlansUrl);
+        results.add(rwtPlansCheckDate == null ? null : rwtPlansCheckDate.toString());
+        results.add(rwtResultsUrl);
+        results.add(rwtResultsCheckDate == null ? null : rwtResultsCheckDate.toString());
```

Comment:
Just curious - does the generated CSV look right when any of the "could be null" values are actually null? I think in the past I've usually put in empty strings instead of null for the CSVs but maybe null works out just fine with the apache CSV library. Approving but I wanted to ask the question!

Reply:
Double checked the CSV and NULLs are output as empty strings.